qiime2/core/cache.py

```diff
@@ -965,9 +976,9 @@ class Pool:
     different piece of data. There are two types of pool:
 
     **Process Pools:** These pools have names of the form
-    <pid>-<create-time>@<uname> based on the process that created them. They
-    only exist for the length of the process that created them and ensure data
-    that process is using stays in the cache.
+    <process-id>-<process-create-time>@$USER based on the process that created
```

Comment:
I would stick with the OG formatting, just use `user` instead of `uname` to be consistent